### PR TITLE
Series.abs() in new style

### PR DESCRIPTION
**Pull request by akharche**

### CODE REVIEW

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 1]
```diff
@@ -1128,3 +1129,35 @@ def hpat_pandas_series_le_impl(self, other):
...
+
+    Returns
+    -------
+    :obj:`pandas.Series`
+         returns :obj:`pandas.Series` containing the absolute value of elements
+    """
+
+    _func_name = 'Method abs().'
+
+    if not isinstance(self, SeriesType):
```
Comment (shssf) [inline hpat/datatypes/hpat_pandas_series_functions.py]: I think we need additional check like this. As I see this method can work with numeric data only but Series can contain, for example, strings (not at this time but in general).
In pandas.abs() documentation we have `This function only applies to elements that are all numeric.`. I think we need to raise exception if underling data is not numeric.

File: hpat/hiframes/pd_series_ext.py [hunk 2]
```diff
@@ -598,14 +598,14 @@ def _resolve_combine_func(self, ary, args, kws):
...
 
-    @bound_function("series.abs")
-    def resolve_abs(self, ary, args, kws):
-        # call np.abs(A) to get return type
-        arr_typ = series_to_array_type(ary)
-        all_args = tuple([arr_typ] + list(args))
-        ret_typ = self.context.resolve_function_type(np.abs, all_args, kws).return_type
-        ret_typ = if_arr_to_series_type(ret_typ)
-        return signature(ret_typ, *args)
+    # @bound_function("series.abs")
```
Comment (shssf) [inline hpat/hiframes/pd_series_ext.py]: This is good. This is typing stage. What happen if comment out definition of the function here https://github.com/IntelPython/hpat/blob/ea6b1a22ba3a04d4495b6a69bb762c09dc41be67/hpat/hiframes/series_kernels.py#L516

File: hpat/datatypes/hpat_pandas_series_functions.py [hunk 3]
```diff
@@ -33,6 +33,7 @@
 
 import operator
 import pandas
+import numpy as np
```
Comment (shssf) [inline hpat/datatypes/hpat_pandas_series_functions.py]: Could you please do not rename numpy?

File: hpat/datatypes/hpat_pandas_series_functions.py (hunk 1 above)
Comment (Rubtsowa) [reply hpat/datatypes/hpat_pandas_series_functions.py]: @shssf In 1135 line "def hpat_pandas_series_append(self):", but it must be   "def hpat_pandas_series_abs(self):".

**Pull request merged**